src/bci_build/package/basecontainers.py

```diff
@@ -199,6 +200,39 @@ def _get_supported_until_fips(os_version: OsVersion) -> datetime.date:
 ]
 
 
+FIPS_MICRO_CONTAINERS = [
+    OsContainer(
+        name="micro-fips",
+        os_version=os_version,
+        support_level=SupportLevel.L3,
+        supported_until=_get_supported_until_fips(os_version),
+        logo_url="https://opensource.suse.com/bci/SLE_BCI_logomark_green.svg",
+        is_latest=os_version in CAN_BE_LATEST_OS_VERSION,
+        pretty_name=_get_fips_pretty_name(os_version),
+        from_target_image=generate_from_image_tag(os_version, "bci-micro"),
+        build_recipe_type=BuildType.DOCKER,
+        package_list=[name for name in ["coreutils", "crypto-policies-scripts", *os_version.release_package_names]
+
+        ],
+        extra_labels={
+            "usage": "This container should only be used on a FIPS enabled host (fips=1 on kernel cmdline)."
+        },
+        custom_end=_get_fips_base_custom_end(os_version)
+        + textwrap.dedent(
+            """
+            ENV GNUTLS_FORCE_FIPS_MODE=1
```

Comment:
this is a perfect copy&paste from `FIPS_BASE_CONTAINERS`. can we separate that out into a helper function (following DRY principle)?